elections/2025/governance-committee-election.md

```diff
@@ -0,0 +1,105 @@
+
+# Nominations
+
+As per [the charter document](../../governance-charter.md#eligibility-for-candidacy), anybody is eligible to run for the Governance Committee. During the "call for nomination" period, people can be nominated or nominate themselves by submitting a Pull Request adding said candidate to the [governance-committee-candidates.md](./governance-committee-candidates.md) file in the OpenTelemetry [community](https://github.com/open-telemetry/community) repository. The template in that file includes the following columns:
+
+* Full name
+* GitHub alias
+* Company affiliation (if applicable)
+* Short bio or reasoning to join the Governance Committee (no more than a short paragraph)
+* _Optional_: photo/picture of a nominee
+
+The election committee will not merge the Pull Request until the candidate has confirmed their desire to be nominated (if not self-nominating) and has been ratified via PR comments.
+
+Nominees should also send their email address to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io) – it will be kept private and used only for candidate communications as the election process proceeds.
+
+The Governance Committee or appointed people will contact every nominee directly to ensure the commitment and desire to be nominated.
+
+# Voter Eligibility
+
+All [members of standing](../../governance-charter.md#members-of-standing) will automatically be eligible to vote. To confirm your eligibility status, see the [election announcements issue](https://github.com/open-telemetry/community/issues/3001). If your code contributions do not meet eligibility requirements, but you believe your non-code contributions should make you eligible to vote, you can request an exemption by submitting an exemption [request form](https://docs.google.com/forms/d/e/1FAIpQLSeSA09xDIv0uyb6vrP8xBbLjm8NsgihrG8GHxacbigF17sNDw/viewform?usp=dialog).
+
+One of two options will be available in the form to prove eligibility:
+
+* GitHub handle
+* List either:
+  * Your contributions to OpenTelemetry that make you eligible to vote, or
+  * Your non-code contributions per the [Members of Standing section of the charter](../../governance-charter.md#members-of-standing): particularly, your relationship to a well-known project that's taken a hard dependency on OpenTelemetry, or your involvement with a well-known organization's adoption of OpenTelemetry as an end-user.
+
+All exemption forms are private. Only the current governance and election committee members will have access to this information. The governance committee will discard the lists one month after the election.
+
+# Vote
+
+Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/elections/f94a7c58-990b-11f0-a16d-5270fb641b4c) using their GitHub account. Voting will be approval voting, where each voter may select up to four candidates. The four candidates with the most votes win the election.
```

Comment:
The wording here sounds a bit weird to me (though I guess it is what has been used in past years). It would sound more natural to me like so
```suggestion
Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/elections/f94a7c58-990b-11f0-a16d-5270fb641b4c) using their GitHub account. Voting happen through approval voting, where each voter may select up to four candidates. The four candidates with the most votes win the election.
```